content/docs/user-guide/how-to/best-practices.md

```diff
@@ -1,16 +1,11 @@
-# Collaboration Issues in Data Science
```

Comment:
what is this page about? it's hidden for now?

Reply:
how to is a strange place for this page?

Reply:
This file was removed along with the entire what-is-dvc/ dir.

Reply:
removing this?

Reply:
Oh I see what you meant now. Similar to the Basic Concepts page, this was moved to become one of the first How-tos as explained in the PR description:

> Happens to start #899 and #72 (create Best Practices How To doc, and section) — hidden (in /doc/user-guide/how-to/best-practices)

But OK I removed this change for now. I'll move it to another branch that addresses that issue separately.